### PR TITLE
camerarecord: add frame stats collection through dumpsys gfxinfo

### PR DESCRIPTION
**Pull request by mdigiorgio**
Signed-off-by: Michele Di Giorgio <michele.digiorgio@arm.com>

### CODE REVIEW

File: wlauto/workloads/camerarecord/__init__.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 #
 
+import os
 from wlauto import UiAutomatorWorkload, Parameter
```
Comment (setrofim) [inline wlauto/workloads/camerarecord/__init__.py]: Please add a space between stdlib and wlauto imports

File: wlauto/workloads/camerarecord/__init__.py [hunk 2]
```diff
@@ -51,8 +52,18 @@ def initialize(self, context):
...
     def setup(self, context):
         super(Camerarecord, self).setup(context)
+        # Start camera activity
         self.device.execute('am start -n {}/{}'.format(self.package, self.activity))
+        # Reset framestats collection
+        self.device.execute('dumpsys gfxinfo {} reset'.format(self.package))
 
     def teardown(self, context):
+        # Collect framestats
+        self.device.execute('dumpsys gfxinfo {} > /data/framestats.txt'.format(self.package))
```
Comment (setrofim) [inline wlauto/workloads/camerarecord/__init__.py]: framestats.txt path should be under self.device.working_directory

**Comment by mdigiorgio** [comment]
Pushed new version with requested changes...

File: wlauto/workloads/camerarecord/__init__.py [hunk 3]
```diff
@@ -51,8 +53,22 @@ def initialize(self, context):
...
     def setup(self, context):
         super(Camerarecord, self).setup(context)
+        # Start camera activity
         self.device.execute('am start -n {}/{}'.format(self.package, self.activity))
+        # Reset framestats collection
+        self.device.execute('dumpsys gfxinfo {} reset'.format(self.package))
 
     def teardown(self, context):
+        # Collect framestats
+        framestats_file = os.path.join(self.device.working_directory,
```
Comment (setrofim) [inline wlauto/workloads/camerarecord/__init__.py]: Should use `self.device.path.join` instead as this is on-target path.

**Pull request merged**